packages/electron-chrome-extensions/src/browser/api/lib/native-messaging-host.ts

```diff
@@ -125,7 +125,6 @@ export class NativeMessagingHost {
   }
 
   destroy() {
-    if (!this.connected) return
```

Comment:
If `destroy()` is called twice, this will still attempt to send an IPC to `this.sender`. I'll fix this up after merging.